docs/src/modules/components/AdDisplay.js

```diff
@@ -1,42 +1,31 @@
 /* eslint react/jsx-no-target-blank: ["error", { allowReferrer: true }] */
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import clsx from 'clsx';
-import { makeStyles } from '@material-ui/styles';
+import { styled } from '@material-ui/core/styles';
 import { adShape } from 'docs/src/modules/components/AdManager';
 import { adStylesObject } from 'docs/src/modules/components/ad.styles';
 
-const hookFactory = (shape) =>
-  makeStyles((theme) => {
+const Root = styled('span', { shouldForwardProp: (prop) => prop !== 'shape' })(
+  ({ theme, shape }) => {
     const styles = adStylesObject[`body-${shape}`](theme);
+
     return {
-      root: {
-        ...styles.root,
-        '& img': styles.img,
-        '& a, & a:hover': styles.a,
-      },
-      imageWrapper: styles.imgWrapper,
-      description: styles.description,
-      poweredby: styles.poweredby,
+      ...styles.root,
+      '& img': styles.img,
+      '& a, & a:hover': styles.a,
+      '& .AdDisplay-imageWrapper': styles.imgWrapper,
+      '& .AdDisplay-description': styles.description,
+      '& .AdDisplay-poweredby': styles.poweredby,
```

Comment:
This does not work. I am not sure if the plan is to remove JSS in this PR or not.

<img width="1307" alt="Screen Shot 2564-07-08 at 19 16 45" src="https://user-images.githubusercontent.com/18292247/124920284-5fc1a180-e021-11eb-847f-44b92cc36fa9.png">

Reply:
What's broken? Yes, the intention is to remove the JSS from these components.

Reply:
https://material-ui.com/components/buttons/

The link is not in blue but the change is.

<img width="1440" alt="Screen Shot 2564-07-09 at 19 55 20" src="https://user-images.githubusercontent.com/18292247/125080954-a41b7300-e0ef-11eb-85a9-233f02cf58bb.png">

Reply:
Ah yes, the `MarkdownElement` component was converted to emotion in https://github.com/mui-org/material-ui/pull/27184. Let me merge that one first, rebase and see if it will be fixed. Good catch!

Reply:
It's fixed after the rebase:

![image](https://user-images.githubusercontent.com/4512430/125099924-da012d00-e0d8-11eb-9452-057043224877.png)